js/addMemberList.js

```diff
@@ -110,51 +106,50 @@ function addMemberList(guild) {
                 !cachedMembers.includes(m.user.id)
         )
         .toJSON();
-    
-	// Display only the first 100 members
-	onlineMembers = onlineMembers.slice(0,100);
-        // Create online label text
-        let container = document.createElement('div');
-        container.id = 'onlineUserList';
-        container.classList.add('roleContainer');
-        listDiv.appendChild(container);
 
-        let name = document.createElement('span');
-        name.classList.add('roleTitle');
-        name.innerText = 'online';
-        container.appendChild(name);
-
-        // Show online users
-        onlineMembers
-            .sort((m1, m2) => m1.id - m2.id)
-            .forEach((m) => {
-                cachedMembers.push(m.id);
-
-                // Create the outer div
-                let outerDiv = document.createElement('div');
-                outerDiv.classList.add('mLOuterDiv');
-                container.appendChild(outerDiv);
-
-                // Make the div for the user
-                let userDiv = document.createElement('div');
-                userDiv.id = m.id;
-                userDiv.classList.add('mLUserDiv');
-                outerDiv.appendChild(userDiv);
-
-                // Add the user icon
-                let icon = document.createElement('img');
-                icon.src = m.displayAvatarURL({ size: 64 });
-                icon.classList.add('mLIcon');
-                userDiv.appendChild(icon);
-
-                // Make the username text
-                let username = document.createElement('p');
-                username.classList.add('mLUsername');
-                username.innerText = m.nickname || m.user.username;
-                username.style.color = m.roles.color?.hexColor || '#8E9297';
-                userDiv.appendChild(username);
-            });
-    
+    // Display only the first 100 members
```

Comment:
Oop yeah I fixed this in vim yesterday.. I guess I didn't make sure it was formatted